docs/mkdocs.yml

```diff
@@ -0,0 +1,67 @@
+# Meta parameters for mkdocs
+docs_dir: src/
+
+# Project information
+site_name: "Simpleflow documentation"
+copyright: "Copyright &copy; 2017 - Botify"
```

Comment:
Hm? Others files have "Copyright 2014 Greg Leclerq" in them, happy to discuss an upgrade if you have opinions on the subject.